profile/README.md

```diff
@@ -1 +1,46 @@
 # Massa
+
+Welcome aboard on the *Massa* organisation. On this profile you will find every repositories related to the development on the **Massa blockchain**.
+
+## Code
+
+- [massa](https://github.com/massalabs/massa): Main code of the Massa blockchain, builds the node and client, holds all the critical code for the architecture.
+- [massa-proto](https://github.com/massalabs/massa-proto): Repository centralizing protobuf bindings for Massa GRPC.
+- [massa-bridge](https://github.com/massalabs/massa-bridge): Blockchain bridge between Massa and any EVM-compatible blockchains.
+- [PeerNet](https://github.com/massalabs/PeerNet): P2P network backend for the Massa blockchain.
+- [stream_limiter](https://github.com/massalabs/stream_limiter): Rust library to implement read/write speed limits on any stream.
+
+## Smart contracts
+
+- [massa-sc-runtime](https://github.com/massalabs/massa-sc-runtime): Sandbox to run the smart-contracts (compiled as WASM) on a Massa node.
+- [massa-as-sdk](https://github.com/massalabs/massa-as-sdk): SDK to build smart-contracts in AssemblyScript. Abstracts the Massa's ABI function.
+- [massa-sc-toolkit](https://github.com/massalabs/massa-sc-toolkit): Provide tools such as `@massalabs/sc-project-initializer` to improve the experience of smart-contracts development.
+- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart-contract to a WASM binary.
+- [massa-sc-playground](https://github.com/massalabs/massa-sc-playground): Playground in AssemblyScript for smart-contracts.
+- [massa-web3](https://github.com/massalabs/massa-web3): Library to help the implementation of Web3 applications on top of the Massa blockchain.
+- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): SDK for smart-contracts in AssemblyScript made for game development.
+- [massa-standards](https://github.com/massalabs/massa-standards): Code implementing standard smart-contracts, such as NFT or ERC-20.
+- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart contract examples.
+- [as](https://github.com/massalabs/as): Utils to write smart-contracts in AssemblyScript.
```

Comment:
```suggestion
- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): An SDK for smart contracts in AssemblyScript, made for game development.
- [massa-standards](https://github.com/massalabs/massa-standards): Code implementing standard smart contracts, such as NFT or ERC-20.
- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart-contract examples.
- [as](https://github.com/massalabs/as): Utils to write smart contracts in AssemblyScript.
```

Reply:
I fixed the language, however I doubt these are good descriptions.
Massa-standards -> is not only made for showing SC-standards like NFT and ERC-20, but they are intended to host all Massalabs standards (check the repo, there's wallet standard apart from SCs).

Reply:
Yeah, I don't know all the repos so I had to do with what it looked like.
Something like "Repository for smart-contracts standards like NFT and ERC-20, and all Massa standards" ?